snet-ui/HowItWorks/index.tsx

```diff
@@ -33,7 +33,7 @@ export default function HowItWorks({ title, steps, blogLink }: Props) {
             <Grid item xs={2} sm={1} md={2}>
               <Box
                 sx={{
-                  bgcolor: "info.light",
+                  bgcolor: "#184FA7",
```

Comment:
don't hardcode color anywhere. 
Always use it from the theme

Reply:
Fixed in the latest commit